### ISSUE TITLE
Enchantment Symbol Reader

### ISSUE THREAD

**Issue by ICouldntUseMyRealNameSoIHaveThis** [Wurst-Imperium/Wurst-Client#946]
Turns all of the Enchanting symbols into their letters.
Here's a video on how to do it without using a resource pack:
https://youtu.be/xhwyv1zMMBk
(Seems pretty easy to implement)

**Comment by Alexander01998** [comment]
Since Minecraft 1.8, enchantments are in plain English anyways.

![](http://hydra-media.cursecdn.com/minecraft.gamepedia.com/9/9c/1.8-enchanting-grid.png?version=52814efa8ea4d800819d9dc2c1aa5579)
